docs/guide/mqtt.md

```diff
@@ -96,7 +96,7 @@ This are the available apis:
 
```

Comment:
I think we need a WIKI or something to collect use cases and the solutions for custom API calls, like sending Notification Reports etc.

Reply:
Yeah I cover some examples under this

Reply:
What do you want to do with this for this PR?

Reply:
I would keep this for another PR. We should just add some examples based on most used functions